File: misc/runScriptExamples/cppEx/hello.cpp
#include<iostream> 
 
using namespace std;
 
/**
 * A basic hello world program in C++
 */
int main(int argc, char **argv) {

  cout << "Hello World" << endl;
  for(int i=0; i<argc; i++) {
    cout << argv[i] << endl;
  }
  return 0;
}

File: misc/runScriptExamples/csEx/Hello.cs
using System;
namespace HelloWorld
{
    /**
     * A basic hello world program in C
     */
    class Hello 
    {
        static void Main(string[] args) 
        {
            Console.WriteLine("Hello World!");
            foreach(string arg in args) {
              Console.WriteLine(arg);
            }
        }
    }
}